### DISCUSSION TITLE
Where is the core tap stored in /opt/homebrew?

### DISCUSSION THREAD

**Discussion by quackduck** [Homebrew/discussions#4587] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.24
ORIGIN: https://github.com/Homebrew/brew
HEAD: 54c8876dc39047c04de15e7d212979ae8d98cf1c
Last commit: 2 days ago
Core tap JSON: 21 Jun 01:23 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_BAT_THEME: OneHalfDark
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: micro
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.41.0 => /opt/homebrew/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Where is the core tap stored in /opt/homebrew? Essentially, I'm trying to find some list of all default formulae on disk. I can see that `brew config` mentions a "Core tap JSON". Is that a file on disk?

**Comment by SMillerDev** [comment]
The core tap isn't stored on disk as a repository anymore like it was before 4.0. The best way to find a formula is at https://formulae.brew.sh if you ask me. Or you could look at https://GitHub.com/Homebrew/Homebrew-core

**Reply by quackduck** [reply]
I see. How does brew know the core formula list then? Does it query some online list?

**Reply by SMillerDev** [reply]
Yes, it queries the api: https://formulae.brew.sh/docs/api/
